markdown/GameGuides/Inside.md

```diff
@@ -0,0 +1,60 @@
+![Inside](Images\inside_header.png "Shot by cHota gAbbar"){.shadowed .autosize}
+
+## Summary
+
+Feature | Supported
+--|--
+Vanilla Photo Mode | No
+Hotsampling | No
+DSR | Yes (through Desktop)
+Custom Aspect Ratios | No
+Reshade | Yes
+Ansel | No
+Graphics API | DirectX 11
+Game version | <font face="Stores"> E </font> <font face="Stores"> S </font> 5.0.4
+
+## Tools
+
+* [Free cam by Abarichello](https://github.com/abarichello/inside-noclip)  
```

Comment:
I am all in for completeness, but since the other camera script seems better overall, shouldn't it be better to just mentioned the one below? It might cause confusion otherwise, especially since this one is mentioned first.

Reply:
Your point make a lot of sense. However, I personally think it's better to include them both because each has features the other doesn't. For eg, the first one allows for translational movement of the camera horizontally and vertically, while the second one allows to rotate the camera in X and Y without keeping the character at the centre. Also, the second's camera works only when the game is paused. while this isn't the case for the first one. So, if someone wants to take a long exposure shot of the game, this would be possible with the first, not the second.

I think, its better to have them both, but reorder them i.e. IDK31 followed by Abarichello.